src/helm/benchmark/run_specs/classic_run_specs.py

```diff
@@ -46,6 +46,7 @@
 from helm.benchmark.runner import get_benchmark_output_path
 from helm.benchmark.scenarios.scenario import ScenarioSpec, get_scenario_cache_path
 from helm.common.hierarchical_logger import hlog, htrack
+from helm.common.gpu_utils import get_torch_device_name
```

Comment:
Could you move the import to the first line inside `get_liveqa_spec()` and `get_medicationqa_spec()`?

The reason is that this transitively imports torch, which is slow... this change is hacky but I can't think of a better way to fix this.